sed/loader/flash/buffer_handler.py

```diff
@@ -173,9 +204,8 @@ def _save_buffer_files(self, force_recreate: bool, debug: bool) -> None:
             if debug:
                 for file_set in file_sets:
                     self._save_buffer_file(file_set)
-                    print(f"Processed {file_set['raw'].stem}")
             else:
-                Parallel(n_jobs=n_cores, verbose=10)(
+                Parallel(n_jobs=-2, verbose=10, prefer="threads")(
```

Comment:
They make processing extremely slow

Reply:
Thanks for pointing out. It's a mistake. I was testing to see if threading makes it faster because joblib is very slow on maxwell if files == num_cores.